assets/data/sponsor.yml

```diff
@@ -1,43 +1,44 @@
-sponsors:
- -  name: "Gold Sponsor"
+sponsor_types:
+  - name:     "Gold"
```

Comment:
To synchronize with changes in #53, this should be "Gold Sponsor" instead of "Gold".